osc-server/src/main/java/org/osc/core/broker/service/persistence/ServiceFunctionChainEntityMgr.java

```diff
@@ -45,5 +48,16 @@ public static void fromEntity(ServiceFunctionChain sfcEntity, ServiceFunctionCha
 	public static ServiceFunctionChain findById(EntityManager em, Long id) {
         return em.find(ServiceFunctionChain.class, id);
     }
+	
+    public static List<ServiceFunctionChain> listServiceFunctionChainsByVirtualSystem(EntityManager em, VirtualSystem vs) {
```

Comment:
why do we need this? cant we just do vs.getServiceFunctionChains()?

is there a significance to sorting by name?

Reply:
For the first comment.  I found it during unit test case and I opted  for querying than getting from vs.sfc list here is explanation for the bug

2.When checking any existing SFC other than the given SFC to bind connected to a SG and particular Tenant(Redundancy chain check), was getting SFC list from Virtual Systems. But in Add or update of SFC entity services we were not updating SFC list in VirtualSystem Entity. So added a query instead of getting from virtualsystem sfc entity list, which is more preferred.


For the Second comment.. sorting by name...nothing in particular..was just going with example code.. will see if I can remove it.

Reply:
removed sorting by name...